nextbike/io/input.py

```diff
@@ -18,7 +18,10 @@ def read_csv(p_filename, p_io_folder, p_sub_folder=""):
     """
     path = os.path.join(get_data_path(), p_io_folder, p_sub_folder, p_filename)
     try:
-        df = pd.read_csv(path, index_col=0)
+        # index_col=0 throws numpy warning,
+        # so we set it manually after reading in the file
+        df = pd.read_csv(path)
+        df.set_index(df.columns[0], inplace=True)
```

Comment:
Holy shit I allways searched were this warning came from :D
nice :+1:

Reply:
🥇 :D